org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/preferences/cleanup/CleanUpMessages.properties

```diff
@@ -218,3 +218,8 @@ JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_PathSeparator=Path se
 JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_JavaVersionProperty=Java Version property
 JavaFeatureTabPage_CheckboxName_ConstantsForSystemProperty_JavaVersionSpecProperty=Java Spec Version property
 
+JavaFeatureTabPage_CheckboxName_ExplicitEncoding=Set explicit encoding
+JavaFeatureTabPage_RadioName_Aggregate_to_UTF8=Insert/replace to make use of UTF-8 instead of platform encoding.\n Try to use single constant per project. 
```

Comment:
I would suggest dropping the third option with the constant.  It looks messy in the cleanup configuration dialog and unless I am missing something, IMO, offers little benefit from the 2nd option to Insert/replace to make use of UTF-8.  If you want, just remove the radio button for now and if desired, it can be added in the future.

Reply:
Done